docs/about/index.html

```diff
@@ -21,7 +21,7 @@
 </a></div><div class="nav-item"><a href="/caliban/about/" aria-current="page" class="nav-link router-link-exact-active router-link-active">
   About
 </a></div><div class="nav-item"><a href="https://github.com/ghostdogpr/caliban" target="_blank" rel="noopener noreferrer" class="nav-link external">
-  Github
+  GitHub
```

Comment:
I'm happy to drop any particular change.

Afaict, this is all just the literal `GitHub` as opposed to a class or anything exciting (unlike `GitLab` where `Gitlab` appeared once in a class-ish thing...)